static/app/app/controller.js

```diff
@@ -59,6 +60,35 @@ var Controller = Marionette.Object.extend({
             spots.fetch().done(callback);
         }
 
+    },
+    join: function(moveId) {
+      var moves = channel.request('entities:moves');
+
+      $.when(moves.fetch()).done(function(){
+        var currentMove = channel.request("entities:move");
+        var moveToJoin = channel.request('entities:moves').find({id: parseInt(moveId)});
+
+        currentMove.set('time', moveToJoin.get('time').format());
+        currentMove.set('spot', moveToJoin.get('spot'));
+
+        if (currentMove.get('user')) {
+          currentMove.save({}, {
+            dataType: 'text',
```

Comment:
I don't think you should have to specify a `dataType` (saving via the form-view uses `Backbone.model.save`, but doesn't require specifying a `dataType`). But if it doesn't work without it, leave it in.